src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -50,6 +50,14 @@ export const Chatbox: React.FC<ChatboxProps> = ({
 
   const [activeOption, setActiveOption] = useState<ChatOption>();
 
+  const closeQuestionOption = () => {
+    setActiveOption(undefined);
+  };
+
+  const isQuestionOptions = useMemo(() => {
+    return ChatMessageOptions.question.name === activeOption?.name;
```

Comment:
Could it be the compare of `activeOption === ChatMessageOptions.question`?

Reply:
https://github.com/sparkletown/sparkle/pull/1424/commits/1230e904a521ba83c1bee2924b7d36f9642858e3

Reply:
https://github.com/sparkletown/sparkle/commit/1230e904a521ba83c1bee2924b7d36f9642858e3